library/std/src/time.rs

```diff
@@ -457,11 +457,15 @@ impl SystemTime {
     ///
     /// # Examples
     ///
-    /// ```
-    /// use std::time::SystemTime;
+    /// ```no_run
+    /// use std::thread::sleep;
+    /// use std::time::{Duration, SystemTime};
     ///
     /// let sys_time = SystemTime::now();
     /// let difference = sys_time.duration_since(sys_time)
+    /// sleep(Duration::from_secs(1));
```

Comment:
I think it would be better to have a time mock here to stimulate the sleep.

The previous line broke the build, missing semi-colon.

I think we should use `thread::sleep` instead here.

Reply:
Do you have an example for a time mock?

Reply:
No. Or we don't have to mock. Just do a `checked_add` to `SystemTime`.

Reply:
Sorry I am too much of a beginner. I don't understand the checked_add, could you give me a hint?

Reply:
Yes, the method you did is correct.